<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
  <meta charset="utf-8">
  <meta name="generator" content="quarto-0.9.17">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Jannik Buhr, Florian Franz, Frauke Gräter">
  <meta name="dcterms.date" content="2021-09-07">
  <title>Intrinsically disordered region of talin’s FERM domain functions as an initial PIP2 recognition site</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      {   }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span.al { color: #ff0000; font-weight: bold; } /* Alert */
    code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
    code span.at { color: #7d9029; } /* Attribute */
    code span.bn { color: #40a070; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #4070a0; } /* Char */
    code span.cn { color: #880000; } /* Constant */
    code span.co { color: #60a0b0; font-style: italic; } /* Comment */
    code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
    code span.do { color: #ba2121; font-style: italic; } /* Documentation */
    code span.dt { color: #902000; } /* DataType */
    code span.dv { color: #40a070; } /* DecVal */
    code span.er { color: #ff0000; font-weight: bold; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #40a070; } /* Float */
    code span.fu { color: #06287e; } /* Function */
    code span.im { } /* Import */
    code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
    code span.kw { color: #007020; font-weight: bold; } /* Keyword */
    code span.op { color: #666666; } /* Operator */
    code span.ot { color: #007020; } /* Other */
    code span.pp { color: #bc7a00; } /* Preprocessor */
    code span.sc { color: #4070a0; } /* SpecialChar */
    code span.ss { color: #bb6688; } /* SpecialString */
    code span.st { color: #4070a0; } /* String */
    code span.va { color: #19177c; } /* Variable */
    code span.vs { color: #4070a0; } /* VerbatimString */
    code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>

  <script src="index_files/libs/clipboard/clipboard.min.js"></script>
  <script src="index_files/libs/quarto-html/quarto.js"></script>
  <script src="index_files/libs/quarto-html/popper.min.js"></script>
  <script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
  <script src="index_files/libs/quarto-html/anchor.min.js"></script>
  <link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
  <link class="quarto-color-scheme" href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet">
  <link class="quarto-color-scheme quarto-color-alternate" rel="prefetch" href="index_files/libs/quarto-html/quarto-syntax-highlighting-dark.css">
  <script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
  <link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
  <link class="quarto-color-scheme" href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet">
  <link class="quarto-color-scheme quarto-color-alternate" rel="prefetch" href="index_files/libs/bootstrap/bootstrap-dark.min.css">
  <script defer="" src="https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.js"></script>
  <script>document.addEventListener("DOMContentLoaded", function () {
 var mathElements = document.getElementsByClassName("math");
 var macros = [];
 for (var i = 0; i < mathElements.length; i++) {
  var texText = mathElements[i].firstChild;
  if (mathElements[i].tagName == "SPAN") {
   katex.render(texText.data, mathElements[i], {
    displayMode: mathElements[i].classList.contains('display'),
    throwOnError: false,
    macros: macros,
    fleqn: false
   });
}}});
  </script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.css">
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc">
<h2 id="toc-title">Table of contents</h2>
<ul>
<li><a href="#introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
<li><a href="#materials-and-methods" class="nav-link" data-scroll-target="#materials-and-methods">Materials and Methods</a>
<ul class="collapse">
<li><a href="#molecular-dynamics-with-gromacs" class="nav-link" data-scroll-target="#molecular-dynamics-with-gromacs">Molecular dynamics with GROMACS</a></li>
<li><a href="#automation-data-analysis-and-availability" class="nav-link" data-scroll-target="#automation-data-analysis-and-availability">Automation, Data Analysis and Availability</a></li>
</ul></li>
<li><a href="#results" class="nav-link" data-scroll-target="#results">Results</a>
<ul class="collapse">
<li><a href="#the-f1-loop-can-act-as-a-point-of-first-contact-todo" class="nav-link" data-scroll-target="#the-f1-loop-can-act-as-a-point-of-first-contact-todo">The F1 loop can act as a point of first contact TODO:</a></li>
<li><a href="#the-f1-loop-can-facilitate-further-contacts-todo" class="nav-link" data-scroll-target="#the-f1-loop-can-facilitate-further-contacts-todo">The F1 loop can facilitate further contacts TODO:</a></li>
<li><a href="#lateral-pulling-of-the-ferm-domain" class="nav-link" data-scroll-target="#lateral-pulling-of-the-ferm-domain">Lateral Pulling of the FERM domain</a></li>
</ul></li>
<li><a href="#discussion" class="nav-link" data-scroll-target="#discussion">Discussion</a></li>
<li><a href="#conclusion" class="nav-link" data-scroll-target="#conclusion">Conclusion</a></li>
<li><a href="#author-contributions" class="nav-link" data-scroll-target="#author-contributions">Author Contributions</a></li>
<li><a href="#acknowledgments" class="nav-link" data-scroll-target="#acknowledgments">Acknowledgments</a></li>
<li><a href="#supplementary-material" class="nav-link" data-scroll-target="#supplementary-material">Supplementary Material</a>
<ul class="collapse">
<li><a href="#trajectories" class="nav-link" data-scroll-target="#trajectories">Trajectories</a></li>
<li><a href="#scripts" class="nav-link" data-scroll-target="#scripts">Scripts</a></li>
</ul></li>
<li><a href="#references" class="nav-link" data-scroll-target="#references">References</a></li>
</ul>
</nav>
</div>
<main class="content page-columns page-full" id="quarto-document-content">
<header id="title-block-header">
<h1 class="title display-7"><p>Intrinsically disordered region of talin’s FERM domain functions as an initial PIP2 recognition site</p></h1>
<p class="author">Jannik Buhr, Florian Franz, Frauke Gräter</p>
<p class="date">2021-09-07</p>
<div class="abstract">
<div class="abstract-title">Abstract</div>
<p>Focal adhesions mediate the interaction of the cytoskeleton with the extracellular matrix (ECM). Cell-ECM adhesion is used by almost all cells both during development and homeostasis and ranges from dynamic to permanent. As such, it is an important process in health and disease alike. Talin is a central regulator and adaptorprotein of the multiprotein focal adhesion complexes and is responsible for integrin activation and force-sensing. We evaluated direct interactions of talin with the membrane lipid phosphatidylinositol 4,5-bisphosphate (PIP<sub>2</sub>) by means of molecular dynamics simulations. A newly published autoinhibitory structure of talin, where common PIP<sub>2</sub> interaction sites are covered up, sparked our curiosity for a hitherto less examined loop as a potential site of first contact. We show that this unstructured loop in the F1 subdomain of the talin1 FERM domain is able to interact with PIP<sub>2</sub> and can facilitate further interactions by serving as a flexible membrane anchor. This work presents the dynamics of the interaction and identifies key residues. Further, we surveyed the effect of a physiological PIP<sub>2</sub> enrichment at focal adhesion sites on the dynamics of talin through force-probe molecular dynamics simulations. The results provide backing for the direct involvement of PIP<sub>2</sub> in the localization and activation of talin.</p>
</div>
</header>

<div class="callout-warning callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Warning
</div>
</div>
<div class="callout-body-container callout-body">
<p>This is a draft and as such subject to change.</p>
</div>
</div>
<div class="callout-tip callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Tip
</div>
</div>
<div class="callout-body-container callout-body">
<p>You can find the poster that goes along with the paper presented at the <a href="https://www.biophysics.org/2022meeting#">Annual Meeting of the Biophysical Society 2022</a> in your favorite format here: <a href="./poster-bps-2022.html">web/html</a>, <a href="./poster-jannik-buhr-bps-2022.pdf">print/pdf</a></p>
</div>
</div>
<section id="introduction" class="level1 page-columns page-full">
<h1>Introduction</h1>
<p>It is critical for cells to mechanically sense their surroundings at cell adhesion sites for a multitude of biological processes. Contact with the extracellular matrix and surroundin cells regulates growth, differentiation, motility and even apoptosis <span class="citation" data-cites="vogelLocalForceGeometry2006 oakesStressingLimitsFocal2014 schillerMechanosensitivityCompositionalDynamics2013 miroshnikovaAdhesionForcesCortical2018">(<a href="#ref-vogelLocalForceGeometry2006" role="doc-biblioref">1</a>–<a href="#ref-miroshnikovaAdhesionForcesCortical2018" role="doc-biblioref">4</a>)</span>. The multiprotein focal adhesion complex is responsible for translating between biochemical and mechanical signals, where both directions, outside–in and inside–out activation, are being investigated <span class="citation" data-cites="thamilselvanPressureActivatesColon2004 pelletierActivationStateIntegrin1995">(<a href="#ref-thamilselvanPressureActivatesColon2004" role="doc-biblioref">5</a>, <a href="#ref-pelletierActivationStateIntegrin1995" role="doc-biblioref">6</a>)</span>.</p>
<p>At the center of the focal adhesion complex sits the adaptorprotein talin, which dynamically unfolds and refolds under force <span class="citation" data-cites="yaoMechanicalResponseTalin2016">(<a href="#ref-yaoMechanicalResponseTalin2016" role="doc-biblioref">7</a>)</span>. A schematic of Talin can be seen in <a href="#fig-tln-schema-long">Figure&nbsp;1 (a)</a>. Through direct interaction with integrin tails (dark green), which interact with collagen fibers via their heads, it links the extracellular matrix to the intracellular cytoskeleton by directly interacting with acting and increasing integrins affinity for ligands, thereby activating it <span class="citation" data-cites="tadokoroTalinBindingIntegrin2003">(<a href="#ref-tadokoroTalinBindingIntegrin2003" role="doc-biblioref">8</a>)</span>.</p>
<div class="column-body-outset quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-tln-schema-long" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/blender/render/frame0000.png" class="img-fluid figure-img" data-ref-parent="fig-structure"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-tln-schema-autoinhib" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/blender/render/frame0001.png" class="img-fluid figure-img" data-ref-parent="fig-structure"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-tln-align-autoinhib" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/blender/render-align/frame0000.png" class="img-fluid figure-img" data-ref-parent="fig-structure"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(c) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 1: <strong>a)</strong> TODO: This is what full-length talin is made up of, these are the important sites, notice integrin binding site and pip2 binding surface. A stylized representation of the Talin1 (sub-)domains over the inner leaflet of a POPC membrane containing PIP<sub>2</sub>. The two main PIP<sub>2</sub>-binding sites are indicated with red dots. Tails of an integrin <span class="math inline">\alpha</span> and <span class="math inline">\beta</span> heterodimer are represented in green. <strong>b)</strong> TODO: Dedden et all found this autoinhibited structure, pip2 binding surface if covered by R12. Additionally R9 covers up Integrin bindin site. The same stylized representation as a), this time depicting the autoinhibited state of Talin. The main PIP<sub>2</sub>-binding sites in F2 and F3 are occluded by rod domain 12 <strong>c)</strong> TODO: Same as b) but with more detail as a cartoon representation. This is the structure found by Dedden et all with the addition of F0 F1, which are not in the initial crystal structure, due to their flexibility. For the complete extended conformation of the FERM domain we used the structure by <span class="citation" data-cites="elliottStructureTalinHead2010">(<a href="#ref-elliottStructureTalinHead2010" role="doc-biblioref">9</a>)</span> <a href="https://www.rcsb.org/structure/3IVF" class="uri">https://www.rcsb.org/structure/3IVF</a>. However, once again due to flexibility it is missing the F1 loop. We modelled with MODELLER and are find the expected flexibility of the loop in simulations using RMSF as a measure, see supplementary <a href="#fig-loop-rmsf">Figure&nbsp;4 (a)</a>.</figcaption><p></p>
</figure>
</div>
</section>
<section id="materials-and-methods" class="level1">
<h1>Materials and Methods</h1>
<section id="molecular-dynamics-with-gromacs" class="level2">
<h2 class="anchored" data-anchor-id="molecular-dynamics-with-gromacs">Molecular dynamics with GROMACS</h2>
<!-- TODO: Add reference scripts for supplementary materials. -->
<p>Molecular dynamics simulations where performed with GROMACS <span class="citation" data-cites="berendsenGROMACSMessagepassingParallel1995 abrahamGROMACSHighPerformance2015">(<a href="#ref-berendsenGROMACSMessagepassingParallel1995" role="doc-biblioref">10</a>, <a href="#ref-abrahamGROMACSHighPerformance2015" role="doc-biblioref">11</a>)</span> version 2020.03 <span class="citation" data-cites="lindahlGROMACS2020Source2020">(<a href="#ref-lindahlGROMACS2020Source2020" role="doc-biblioref">12</a>)</span>. A crystal structure of the Talin FERM domain by Goult et al. <span class="citation" data-cites="elliottStructureTalinHead2010">(<a href="#ref-elliottStructureTalinHead2010" role="doc-biblioref">9</a>)</span> with the PDB-ID 3IVF was used as the basis of all simulations.</p>
<!-- :::video-container -->
<!-- [FERM]{#app1 .app} -->
<!-- ::: -->
<p>The missing F1 domain loop between residues L133 and W144 was modeled using MODELLER <span class="citation" data-cites="marti-renomComparativeProteinStructure2000 webbComparativeProteinStructure2016">(<a href="#ref-marti-renomComparativeProteinStructure2000" role="doc-biblioref">13</a>, <a href="#ref-webbComparativeProteinStructure2016" role="doc-biblioref">14</a>)</span> via the interface to Chimera <span class="citation" data-cites="pettersenUCSFChimeraVisualization2004">(<a href="#ref-pettersenUCSFChimeraVisualization2004" role="doc-biblioref">15</a>)</span>, followed by equilibration with GROMACS. The resulting conformation was compared to an NMR structure of the F1 domain (PDB-ID 2KC2) by Goult et al. <span class="citation" data-cites="goultStructureDoubleUbiquitinlike2010">(<a href="#ref-goultStructureDoubleUbiquitinlike2010" role="doc-biblioref">16</a>)</span>.</p>
<p>The missing residue M1 was also added. The missing residues I399 and L400 where not modeled, leaving us with with a continuous sequence from residue 1 to 398. Simulations where performed with the CHARM36 force field. Topologies, including the membrane, where generated with the CHARM-GUI web app <span class="citation" data-cites="brooksCHARMMBiomolecularSimulation2009 joCHARMMGUIWebbasedGraphical2008 leeCHARMMGUIInputGenerator2016">(<a href="#ref-brooksCHARMMBiomolecularSimulation2009" role="doc-biblioref">17</a>–<a href="#ref-leeCHARMMGUIInputGenerator2016" role="doc-biblioref">19</a>)</span> and GROMACS tools. All simulations used the TIP3P water model and where neutralized with 0.15 mol/L of NaCl. A a 6-step equilibration was performed after gradient decent energy minimization while gradually relieving restraints on protein and membrane atoms. Production runs use a timestep of 2 fs, a Verlet cut-off scheme for Van-der-Waals interactions and the Particle Mesh Ewald (PME) method for long-range electrostatics. NTP-ensembles where achieved by Nosé-Hoover temperature coupling <span class="citation" data-cites="hooverCanonicalDynamicsEquilibrium1985 noseUnifiedFormulationConstant1984">(<a href="#ref-hooverCanonicalDynamicsEquilibrium1985" role="doc-biblioref">20</a>, <a href="#ref-noseUnifiedFormulationConstant1984" role="doc-biblioref">21</a>)</span> and Parinello-Rahman pressure coupling <span class="citation" data-cites="parrinelloPolymorphicTransitionsSingle1981">(<a href="#ref-parrinelloPolymorphicTransitionsSingle1981" role="doc-biblioref">22</a>)</span>. An example <code>.mdp</code>-file can be found in the supplementary materials at <a href="#sec-prod-mdp">Section&nbsp;8.2.1</a>.</p>
<p>The initial equilibrium simulation of the completed FERM domain was run for 75 ns. Subsequently, the root mean squared fluctuation (RMSF) was calculated with GROMACS tools.</p>
<p>The F0F1 FERM sub domains (residues 1 to 197) were simulated to evaluate protein-membrane association using a rotational sampling approach. This entailed placing the protein 1.5 nm away from a 1-palmitoyl-2-oleoyl-glycero-3-phosphocholine (POPC) membrane in a total of 60 orientations spanning a rotation of 360 degrees. 6 replicates of each orientation where run for 200&nbsp;ns each. However, due to a hardware failure, 6 of these 360 runs are only 50 to 150&nbsp;ns long. Of the 119 lipids in the upper leaflet of the POPC membrane, 12 lipids were replaced with PIP<sub>2</sub>, which results in a physiological concentration of 10% PIP<sub>2</sub>.</p>
<p>From this rotational sampling, we selected representative conformations with high protein-membrane interaction as the basis of 6 equilibrium simulations of the complete FERM domain over a POPC membrane with 26 PIP<sub>2</sub> lipids out of a total of 273 lipids in the upper leaflet. Each simulation ran for 400&nbsp;ns. The initial conformations for perpendicular pulling simulations of the F0F1 subdomains to gauge interaction strength where also chose from the rotational sampling set.</p>
<p>To compare the effects of PIP<sub>2</sub> on the dynamics, a system containing only 1&nbsp;%&nbsp;PIP<sub>2</sub> (3 PIP<sub>2</sub> molecules in 273 lipids) was created from the enriched system by programmatically exchanging lipid head groups via a python script. The enriched and non-enriched systems where then subjected to force-probe MD simulations.</p>
</section>
<section id="automation-data-analysis-and-availability" class="level2">
<h2 class="anchored" data-anchor-id="automation-data-analysis-and-availability">Automation, Data Analysis and Availability</h2>
<p>Setup scripts written in bash are available for all simulations shown in this work. Plots were generated with R using ggplot2 <span class="citation" data-cites="ggplot">(<a href="#ref-ggplot" role="doc-biblioref">23</a>)</span>. Interactive structure representations are embedded using Mol* <span class="citation" data-cites="molstar">(<a href="#ref-molstar" role="doc-biblioref"><strong>molstar?</strong></a>)</span>. Schematic visualizations were rendered with blender <span class="citation" data-cites="blender">(<a href="#ref-blender" role="doc-biblioref">24</a>)</span> and VMD <span class="citation" data-cites="vmd">(<a href="#ref-vmd" role="doc-biblioref">25</a>)</span>. Files relevant to this paper that are too big to be uploaded to this repository, such as trajectories and blender files, will be uploaded to a separate location.</p>
<p>This paper and the matching poster were generated with <a href="https://quarto.org/">quarto</a> <span class="citation" data-cites="quarto">(<a href="#ref-quarto" role="doc-biblioref">26</a>)</span> and Rmarkdown <span class="citation" data-cites="knitr rbetterposter">(<a href="#ref-knitr" role="doc-biblioref">27</a>, <a href="#ref-rbetterposter" role="doc-biblioref">28</a>)</span>.</p>
<!-- TODO: upload trajectories -->
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<section id="the-f1-loop-can-act-as-a-point-of-first-contact-todo" class="level2">
<h2 class="anchored" data-anchor-id="the-f1-loop-can-act-as-a-point-of-first-contact-todo">The F1 loop can act as a point of first contact TODO:</h2>
<div class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 33.3%;justify-content: center;">
<div id="fig-f0f1-unbound" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/vmd/f0f1/unbound.png" class="img-fluid figure-img" data-ref-parent="fig-loop-importance"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 33.3%;justify-content: center;">
<div id="fig-f0f1-anchored" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/vmd/f0f1/anchored.png" class="img-fluid figure-img" data-ref-parent="fig-loop-importance"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 33.3%;justify-content: center;">
<div id="fig-f0f1-bound" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/vmd/f0f1/bound.png" class="img-fluid figure-img" data-ref-parent="fig-loop-importance"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(c) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-f0f1-ri-angle" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/f0f1-ri-angle-npip-1.png" class="img-fluid figure-img" data-ref-parent="fig-loop-importance"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(d) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-f0f1-angle-frame" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/f0f1-angle-frame-npip-1.png" class="img-fluid figure-img" data-ref-parent="fig-loop-importance"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(e) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-f0f1-retention" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/f0f1-retention-1.png" class="img-fluid figure-img" data-ref-parent="fig-loop-importance"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(f) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-f0f1-ri-npip" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/f0f1-ri-npip-1.png" class="img-fluid figure-img" data-ref-parent="fig-loop-importance"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(g) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><video src="./assets/vmd/f0f1/residues.mp4" class="img-fluid" controls=""><a href="./assets/vmd/f0f1/residues.mp4">Video</a></video></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">&nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 2: <strong>a)</strong> TODO: The F1-loop interacts with PIP<sub>2</sub> and can anchor the subdomain to the membrane due to its high flexibility. But in order to test if this interaction of the loop with the membrane is just the result of a biased starting position with the loop already pointing downwards we sampled 60 different starting positions, rotating around the x-axis, with 6 replicates each. <strong>a)</strong> TODO: Regardless of the angle the loop is almost always involved in the contact residues <strong>b)</strong> TODO: And angles in which the loop is already favored towards the membrane tend to make contact faster. A heatmap of the first 100 ns of all 360 simulation s, summarized per angle, showing the average number of residues in contact with at least one PIP<sub>2</sub> for that frame. The F1-loop finds the membrane faster than other parts of F0F1 can due to it’s high flexibility. <strong>c)</strong> TODO: f0f1 retention <strong>d)</strong> TODO: Mean interaction scores of the individual residues across all 6 simulations. Color represents the PI of the amino acid in isolation. A number of very prominent lysines can be observed.</figcaption><p></p>
</figure>
</div>
</section>
<section id="the-f1-loop-can-facilitate-further-contacts-todo" class="level2">
<h2 class="anchored" data-anchor-id="the-f1-loop-can-facilitate-further-contacts-todo">The F1 loop can facilitate further contacts TODO:</h2>
<div class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><video src="./assets/vmd/f0f1-pulling/f0f1-run1.mp4" class="img-fluid" controls=""><a href="./assets/vmd/f0f1-pulling/f0f1-run1.mp4">Video</a></video></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">&nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 43.5%;justify-content: center;">
<div id="fig-f0f1-vert-pull-vmd" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/vmd/f0f1-pulling/vert-pull.png" class="img-fluid figure-img" data-ref-parent="fig-loop-further-interactions"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 56.5%;justify-content: center;">
<div id="fig-f0f1-vert-pull" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/f0f1-vert-pull-1.png" class="img-fluid figure-img" data-ref-parent="fig-loop-further-interactions"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-ferm-ri-npip" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/ferm-ri-npip-1.png" class="img-fluid figure-img" data-ref-parent="fig-loop-further-interactions"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(c) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-ferm-memb-system" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/vmd/ferm/ferm-residues-transparent-arrows.png" class="img-fluid figure-img" data-ref-parent="fig-loop-further-interactions"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(d) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 3: <strong>a)</strong> TODO: <strong>b)</strong> TODO: see K272 of F2 and K316, K324, E342, and K343 of F3 <span class="citation" data-cites="chinthalapudiInteractionTalinCell2018a">(<a href="#ref-chinthalapudiInteractionTalinCell2018a" role="doc-biblioref">29</a>)</span>. <strong>c)</strong> TODO: The complete simulation system with the talin1 FERM domain over a POPC membrane containing 10&nbsp;% PIP<sub>2</sub> in the upper leaflet (26 PIP2 molecules for simulations with the full-length ferm domain, 12 for F0F1 simulations).</figcaption><p></p>
</figure>
</div>
</section>
<section id="lateral-pulling-of-the-ferm-domain" class="level2">
<h2 class="anchored" data-anchor-id="lateral-pulling-of-the-ferm-domain">Lateral Pulling of the FERM domain</h2>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>Additional simulations pulling the whole FERM domain at a physiological angle of 30° laterally over a membrane with differing concentrations of PIP<sub>2</sub> are currently running.</p>
</div>
</div>
<!-- - force probing / lateral pulling (compare to vertical only-loop pulling) -->
<!--   - many interactions important vs. single strong site -->
<!--   - loop itself only recognition -->
<!-- :::: {#fig-pulling layout="[[1],[1,1]]"} -->
<!---->
<!---->
<!-- ![ ](./assets/results/plots/tln-ferm-lat-pull-1.png){#fig-ferm-lat-pull} -->
<!---->
<!-- ![ ](./assets/results/plots/tln-ferm-lat-pull_mean_se-1.png){#fig-ferm-lat-pull-mean-se} -->
<!---->
<!---->
<!-- **a\)** TODO: -->
<!-- **b\)** TODO: -->
<!-- **c\)** TODO: -->
<!-- :::: -->
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
<!-- TODO: -->
</section>
<section id="conclusion" class="level1">
<h1>Conclusion</h1>
<!-- TODO: -->
</section>
<section id="author-contributions" class="level1">
<h1>Author Contributions</h1>
<p>Conceived and designed the experiments: JB FF FG. Performed the experiments: JB FF. Analyzed the data: JB FF. Contributed reagents/materials/analysis tools: FF. Wrote the paper: JB.</p>
<!-- TODO: -->
</section>
<section id="acknowledgments" class="level1">
<h1>Acknowledgments</h1>
<p>This project has received funding from the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement No.&nbsp;101002812).</p>
<!-- TODO: -->
</section>
<section id="supplementary-material" class="level1">
<h1>Supplementary Material</h1>
<!-- TODO: -->
<!-- ![ ](./assets/results/plots/ferm-time-ri-npip-single-run-1.png){#fig-ferm-time-ri-npip-single} -->
<div class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-loop-rmsf" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/figures/loop-rmsf.png" class="img-fluid figure-img" data-ref-parent="fig-suppl"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-r-hist" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/rot-sample-r-hist-1.png" class="img-fluid figure-img" data-ref-parent="fig-suppl"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-ferm-time-ri-npip-all" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="./assets/results/plots/ferm-time-ri-npip-all-1.png" class="img-fluid figure-img" data-ref-parent="fig-suppl"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(c) &nbsp;</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-top">
<div id="tbl-ferm-domains" class="quarto-layout-cell anchored" style="flex-basis: 33.3%;justify-content: center;">
<table class="table">
<caption>Table 1: Talin1 FERM domain boundaries</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Domain</th>
<th style="text-align: left;">From</th>
<th style="text-align: left;">To</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">F0</td>
<td style="text-align: left;">1</td>
<td style="text-align: left;">84</td>
</tr>
<tr class="even">
<td style="text-align: left;">F1</td>
<td style="text-align: left;">85</td>
<td style="text-align: left;">207</td>
</tr>
<tr class="odd">
<td style="text-align: left;">F2</td>
<td style="text-align: left;">208</td>
<td style="text-align: left;">309</td>
</tr>
<tr class="even">
<td style="text-align: left;">F3</td>
<td style="text-align: left;">310</td>
<td style="text-align: left;">400</td>
</tr>
</tbody>
</table>
</div>
<div id="tbl-f0f1-top-interacting" class="quarto-layout-cell anchored" style="flex-basis: 33.3%;justify-content: center;">
<table class="table">
<caption>Table 2: Top residues interacting with F0F1</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Residue</th>
<th style="text-align: right;">Mean #PIP<sub>2</sub></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">M 1</td>
<td style="text-align: right;">0.188</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 15</td>
<td style="text-align: right;">0.184</td>
</tr>
<tr class="odd">
<td style="text-align: left;">R 30</td>
<td style="text-align: right;">0.173</td>
</tr>
<tr class="even">
<td style="text-align: left;">R 35</td>
<td style="text-align: right;">0.245</td>
</tr>
<tr class="odd">
<td style="text-align: left;">R 74</td>
<td style="text-align: right;">0.124</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 98</td>
<td style="text-align: right;">0.176</td>
</tr>
<tr class="odd">
<td style="text-align: left;">R 118</td>
<td style="text-align: right;">0.209</td>
</tr>
<tr class="even">
<td style="text-align: left;">T 144</td>
<td style="text-align: right;">0.299</td>
</tr>
<tr class="odd">
<td style="text-align: left;">L 145</td>
<td style="text-align: right;">0.168</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 147</td>
<td style="text-align: right;">0.263</td>
</tr>
<tr class="odd">
<td style="text-align: left;">L 151</td>
<td style="text-align: right;">0.325</td>
</tr>
<tr class="even">
<td style="text-align: left;">D 154</td>
<td style="text-align: right;">0.248</td>
</tr>
<tr class="odd">
<td style="text-align: left;">E 155</td>
<td style="text-align: right;">0.261</td>
</tr>
<tr class="even">
<td style="text-align: left;">M 158</td>
<td style="text-align: right;">0.272</td>
</tr>
<tr class="odd">
<td style="text-align: left;">K 160</td>
<td style="text-align: right;">0.254</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 162</td>
<td style="text-align: right;">0.181</td>
</tr>
<tr class="odd">
<td style="text-align: left;">L 193</td>
<td style="text-align: right;">0.200</td>
</tr>
<tr class="even">
<td style="text-align: left;">R 194</td>
<td style="text-align: right;">0.101</td>
</tr>
</tbody>
</table>
</div>
<div id="tbl-ferm-top-interacting" class="quarto-layout-cell anchored" style="flex-basis: 33.3%;justify-content: center;">
<table class="table">
<caption>Table 3: Top residues interacting with FERM</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Residue</th>
<th style="text-align: right;">Mean #PIP<sub>2</sub></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">M 1</td>
<td style="text-align: right;">0.118</td>
</tr>
<tr class="even">
<td style="text-align: left;">T 144</td>
<td style="text-align: right;">0.322</td>
</tr>
<tr class="odd">
<td style="text-align: left;">L 145</td>
<td style="text-align: right;">0.129</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 147</td>
<td style="text-align: right;">0.412</td>
</tr>
<tr class="odd">
<td style="text-align: left;">L 151</td>
<td style="text-align: right;">0.204</td>
</tr>
<tr class="even">
<td style="text-align: left;">D 154</td>
<td style="text-align: right;">0.293</td>
</tr>
<tr class="odd">
<td style="text-align: left;">E 155</td>
<td style="text-align: right;">0.257</td>
</tr>
<tr class="even">
<td style="text-align: left;">M 158</td>
<td style="text-align: right;">0.246</td>
</tr>
<tr class="odd">
<td style="text-align: left;">K 160</td>
<td style="text-align: right;">0.459</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 162</td>
<td style="text-align: right;">0.195</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Y 270</td>
<td style="text-align: right;">0.156</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 272</td>
<td style="text-align: right;">0.180</td>
</tr>
<tr class="odd">
<td style="text-align: left;">G 275</td>
<td style="text-align: right;">0.222</td>
</tr>
<tr class="even">
<td style="text-align: left;">L 314</td>
<td style="text-align: right;">0.141</td>
</tr>
<tr class="odd">
<td style="text-align: left;">K 316</td>
<td style="text-align: right;">0.304</td>
</tr>
<tr class="even">
<td style="text-align: left;">K 318</td>
<td style="text-align: right;">0.148</td>
</tr>
<tr class="odd">
<td style="text-align: left;">K 320</td>
<td style="text-align: right;">0.552</td>
</tr>
<tr class="even">
<td style="text-align: left;">G 321</td>
<td style="text-align: right;">0.174</td>
</tr>
<tr class="odd">
<td style="text-align: left;">K 322</td>
<td style="text-align: right;">0.442</td>
</tr>
<tr class="even">
<td style="text-align: left;">D 341</td>
<td style="text-align: right;">0.142</td>
</tr>
<tr class="odd">
<td style="text-align: left;">S 362</td>
<td style="text-align: right;">0.363</td>
</tr>
</tbody>
</table>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 4: <strong>a)</strong> RMSF [nm] of the c<span class="math inline">\alpha</span> of individual residues in an equilibrium simulation shown by coloring the backbone. The loop is highly flexible. <strong>b)</strong> A density plot of distances between PIP<sub>2</sub> and the protein residues to decide on a cutoff for defining interactions. A distance of 0.25 nm was chosen.</figcaption><p></p>
</figure>
</div>
<section id="trajectories" class="level2">
<h2 class="anchored" data-anchor-id="trajectories">Trajectories</h2>
<!-- :::video-container -->
<!-- [Tln1 trajectory]{#app2 .app} -->
<!-- ::: -->
</section>
<section id="scripts" class="level2">
<h2 class="anchored" data-anchor-id="scripts">Scripts</h2>
<section id="sec-prod-mdp" class="level3">
<h3 class="anchored" data-anchor-id="sec-prod-mdp">Production run parameters</h3>
<div class="sourceCode" id="cb1"><pre class="sourceCode bash code-with-copy"><code class="sourceCode bash"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="ex">integrator</span>              = md</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="ex">dt</span>                      = 0.002</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="ex">nsteps</span>                  = 100000000</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="ex">nstxout</span>                 = 5000</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="ex">nstvout</span>                 = 5000</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="ex">nstfout</span>                 = 50000</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="ex">nstcalcenergy</span>           = 100</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="ex">nstenergy</span>               = 1000</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="ex">nstlog</span>                  = 1000</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="ex">cutoff-scheme</span>           = Verlet</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="ex">nstlist</span>                 = 20</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="ex">rlist</span>                   = 1.2</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="ex">coulombtype</span>             = pme</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="ex">rcoulomb</span>                = 1.2</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="ex">vdwtype</span>                 = Cut-off</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="ex">vdw-modifier</span>            = Force-switch</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="ex">rvdw_switch</span>             = 1.0</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="ex">rvdw</span>                    = 1.2</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="ex">tcoupl</span>                  = Nose-Hoover</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="ex">tc_grps</span>                 = SYSTEM</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="ex">tau_t</span>                   = 1.0</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="ex">ref_t</span>                   = 303.15</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a><span class="ex">pcoupl</span>                  = Parrinello-Rahman</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="ex">pcoupltype</span>              = semiisotropic</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="ex">tau_p</span>                   = 5.0</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="ex">compressibility</span>         = 4.5e-5  4.5e-5</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="ex">ref_p</span>                   = 1.0     1.0</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="ex">constraints</span>             = h-bonds</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="ex">constraint_algorithm</span>    = LINCS</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="ex">continuation</span>            = yes</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="ex">nstcomm</span>                 = 100</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a><span class="ex">comm_mode</span>               = linear</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a><span class="ex">comm_grps</span>               = SYSTEM</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a><span class="ex">refcoord_scaling</span>        = com</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</section>
</section>
</section>
<section id="references" class="level1 unnumbered">
<h1 class="unnumbered">References</h1>
<!-- ```{=html} -->
<!-- <script type="text/javascript" src="./www/molstar.js"></script> -->
<!-- <link rel="stylesheet" type="text/css" href="./www/molstar.css" /> -->
<!-- <script type="text/javascript"> -->
<!-- var viewer1 = new molstar.Viewer('app1', { -->
<!-- layoutIsExpanded: false, -->
<!-- layoutShowControls: false, -->
<!-- layoutShowRemoteState: false, -->
<!-- layoutShowSequence: true, -->
<!-- layoutShowLog: false, -->
<!-- layoutShowLeftPanel: true, -->
<!-- viewportShowExpand: true, -->
<!-- viewportShowSelectionMode: false, -->
<!-- viewportShowAnimation: false, -->
<!-- pdbProvider: "rcsb", -->
<!-- emdbProvider: "rcsb", -->
<!-- }); -->
<!-- // viewer1.loadPdb("3IVF"); -->
<!-- viewer1.loadStructureFromUrl('www/3IVF.pdb', format='pdb') -->
<!-- </script> -->
<!-- ``` -->
<!-- ```{=html} -->
<!-- <script type="text/javascript" src="./www/molstar.js"></script> -->
<!-- <link rel="stylesheet" type="text/css" href="./www/molstar.css" /> -->
<!-- <script type="text/javascript"> -->
<!-- var viewer2 = new molstar.Viewer('app2', { -->
<!-- layoutIsExpanded: false, -->
<!-- layoutShowControls: false, -->
<!-- layoutShowRemoteState: false, -->
<!-- layoutShowSequence: true, -->
<!-- layoutShowLog: false, -->
<!-- layoutShowLeftPanel: true, -->
<!-- viewportShowExpand: true, -->
<!-- viewportShowSelectionMode: false, -->
<!-- viewportShowAnimation: true, -->
<!-- pdbProvider: "rcsb", -->
<!-- emdbProvider: "rcsb", -->
<!-- }); -->
<!-- viewer2.loadTrajectory({ -->
<!--     model: { kind: 'model-url', url: './www/f0f1_rot_sample_center.gro', format: 'gro' }, -->
<!--     coordinates: { kind: 'coordinates-url', url: './www/f0f1_rot_sample_run5_angl150.xtc', format: 'xtc', isBinary: true } -->
<!-- }); -->
<!-- anim = viewer2.plugin.managers.animation.animations[0] -->
<!-- viewer2.plugin.managers.animation.play(anim, { duration: { name: 'computed', params: { targetFps: 30 } } }) -->
<!-- </script> -->
<!-- ``` -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-vogelLocalForceGeometry2006" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Vogel, V., and M. Sheetz. 2006. <a href="https://doi.org/10.1038/nrm1890">Local force and geometry sensing regulate cell functions</a>. <em>Nature Reviews Molecular Cell Biology</em>. 7:265–275.</div>
</div>
<div id="ref-oakesStressingLimitsFocal2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Oakes, P.W., and M.L. Gardel. 2014. <a href="https://doi.org/10.1016/j.ceb.2014.06.003">Stressing the limits of focal adhesion mechanosensitivity</a>. <em>Current Opinion in Cell Biology</em>. 30:68–73.</div>
</div>
<div id="ref-schillerMechanosensitivityCompositionalDynamics2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Schiller, H.B., and R. Fässler. 2013. <a href="https://doi.org/10.1038/embor.2013.49">Mechanosensitivity and compositional dynamics of cellmatrix adhesions</a>. <em>EMBO reports</em>. 14:509–519.</div>
</div>
<div id="ref-miroshnikovaAdhesionForcesCortical2018" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Miroshnikova, Y.A., H.Q. Le, D. Schneider, T. Thalheim, M. Rübsam, N. Bremicker, J. Polleux, N. Kamprad, M. Tarantola, I. Wang, M. Balland, C.M. Niessen, J. Galle, and S.A. Wickström. 2018. <a href="https://doi.org/10.1038/s41556-017-0005-z">Adhesion forces and cortical tension couple cell proliferation and differentiation to drive epidermal stratification</a>. <em>Nature Cell Biology</em>. 20:69–80.</div>
</div>
<div id="ref-thamilselvanPressureActivatesColon2004" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Thamilselvan, V., and M.D. Basson. 2004. <a href="https://doi.org/10.1053/j.gastro.2003.10.078">Pressure activates colon cancer cell adhesion by inside-out focal adhesion complex and actin cytoskeletal signaling</a>. <em>Gastroenterology</em>. 126:8–18.</div>
</div>
<div id="ref-pelletierActivationStateIntegrin1995" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Pelletier, A.J., T. Kunicki, Z.M. Ruggeri, and V. Quaranta. 1995. <a href="https://doi.org/10.1074/jbc.270.30.18133">The <span>Activation State</span> of the <span class="nocase">Integrin <span class="math inline">\alpha</span>IIb<span class="math inline">\beta</span>3 Affects Outside-in Signals Leading</span> to <span>Cell Spreading</span> and <span>Focal Adhesion Kinase Phosphorylation</span> *</a>. <em>Journal of Biological Chemistry</em>. 270:18133–18140.</div>
</div>
<div id="ref-yaoMechanicalResponseTalin2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Yao, M., B.T. Goult, B. Klapholz, X. Hu, C.P. Toseland, Y. Guo, P. Cong, M.P. Sheetz, and J. Yan. 2016. <a href="https://doi.org/10.1038/ncomms11966">The mechanical response of talin</a>. <em>Nature Communications</em>. 7:11966.</div>
</div>
<div id="ref-tadokoroTalinBindingIntegrin2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Tadokoro, S., S.J. Shattil, K. Eto, V. Tai, R.C. Liddington, J.M. de Pereda, M.H. Ginsberg, and D.A. Calderwood. 2003. <a href="https://doi.org/10.1126/science.1086652">Talin <span>Binding</span> to <span>Integrin</span> ß <span>Tails</span>: <span>A Final Common Step</span> in <span>Integrin Activation</span></a>. <em>Science</em>. 302:103–106.</div>
</div>
<div id="ref-elliottStructureTalinHead2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Elliott, P.R., B.T. Goult, P.M. Kopp, N. Bate, J.G. Grossmann, G.C.K. Roberts, D.R. Critchley, and I.L. Barsukov. 2010. <a href="https://doi.org/10.1016/j.str.2010.07.011">The <span>Structure</span> of the <span>Talin Head Reveals</span> a <span>Novel&nbsp;Extended Conformation</span> of the <span>FERM Domain</span></a>. <em>Structure(London, England:1993)</em>. 18:1289–1299.</div>
</div>
<div id="ref-berendsenGROMACSMessagepassingParallel1995" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Berendsen, H.J.C., D. van der Spoel, and R. van Drunen. 1995. <a href="https://doi.org/10.1016/0010-4655(95)00042-E"><span>GROMACS</span>: <span>A</span> message-passing parallel molecular dynamics implementation</a>. <em>Computer Physics Communications</em>. 91:43–56.</div>
</div>
<div id="ref-abrahamGROMACSHighPerformance2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Abraham, M.J., T. Murtola, R. Schulz, S. Páll, J.C. Smith, B. Hess, and E. Lindahl. 2015. <a href="https://doi.org/10.1016/j.softx.2015.06.001"><span>GROMACS</span>: <span>High</span> performance molecular simulations through multi-level parallelism from laptops to supercomputers</a>. <em>SoftwareX</em>. 1–2:19–25.</div>
</div>
<div id="ref-lindahlGROMACS2020Source2020" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Lindahl, Abraham, Hess, and van der Spoel. 2020-01-01, 2020-01. <a href="https://doi.org/10.5281/zenodo.3562495"><span>GROMACS</span> 2020 source code</a>. <span>Zenodo</span>.</div>
</div>
<div id="ref-marti-renomComparativeProteinStructure2000" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Martí-Renom, M.A., A.C. Stuart, A. Fiser, R. Sánchez, F. Melo, and A. Sali. 2000. <a href="https://doi.org/10.1146/annurev.biophys.29.1.291">Comparative protein structure modeling of genes and genomes</a>. <em>Annual Review of Biophysics and Biomolecular Structure</em>. 29:291–325.</div>
</div>
<div id="ref-webbComparativeProteinStructure2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Webb, B., and A. Sali. 2016. <a href="https://doi.org/10.1002/cpps.20">Comparative <span>Protein Structure Modeling Using MODELLER</span></a>. <em>Current Protocols in Protein Science</em>. 86:2.9.1–2.9.37.</div>
</div>
<div id="ref-pettersenUCSFChimeraVisualization2004" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Pettersen, E.F., T.D. Goddard, C.C. Huang, G.S. Couch, D.M. Greenblatt, E.C. Meng, and T.E. Ferrin. 2004. <a href="https://doi.org/10.1002/jcc.20084"><span class="nocase">UCSF Chimera–a</span> visualization system for exploratory research and analysis</a>. <em>Journal of Computational Chemistry</em>. 25:1605–1612.</div>
</div>
<div id="ref-goultStructureDoubleUbiquitinlike2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Goult, B.T., M. Bouaouina, P.R. Elliott, N. Bate, B. Patel, A.R. Gingras, J.G. Grossmann, G.C.K. Roberts, D.A. Calderwood, D.R. Critchley, and I.L. Barsukov. 2010. <a href="https://doi.org/10.1038/emboj.2010.4">Structure of a double ubiquitin-like domain in the talin head: A role in integrin activation</a>. <em>The EMBO Journal</em>. 29:1069–1080.</div>
</div>
<div id="ref-brooksCHARMMBiomolecularSimulation2009" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Brooks, B.R., C.L. Brooks, A.D. Mackerell, L. Nilsson, R.J. Petrella, B. Roux, Y. Won, G. Archontis, C. Bartels, S. Boresch, A. Caflisch, L. Caves, Q. Cui, A.R. Dinner, M. Feig, S. Fischer, J. Gao, M. Hodoscek, W. Im, K. Kuczera, T. Lazaridis, J. Ma, V. Ovchinnikov, E. Paci, R.W. Pastor, C.B. Post, J.Z. Pu, M. Schaefer, B. Tidor, R.M. Venable, H.L. Woodcock, X. Wu, W. Yang, D.M. York, and M. Karplus. 2009. <a href="https://doi.org/10.1002/jcc.21287"><span>CHARMM</span>: <span>The</span> biomolecular simulation program</a>. <em>Journal of Computational Chemistry</em>. 30:1545–1614.</div>
</div>
<div id="ref-joCHARMMGUIWebbasedGraphical2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Jo, S., T. Kim, V.G. Iyer, and W. Im. 2008. <a href="https://doi.org/10.1002/jcc.20945"><span>CHARMM-GUI</span>: A web-based graphical user interface for <span>CHARMM</span></a>. <em>Journal of Computational Chemistry</em>. 29:1859–1865.</div>
</div>
<div id="ref-leeCHARMMGUIInputGenerator2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Lee, J., X. Cheng, J.M. Swails, M.S. Yeom, P.K. Eastman, J.A. Lemkul, S. Wei, J. Buckner, J.C. Jeong, Y. Qi, S. Jo, V.S. Pande, D.A. Case, C.L. Brooks, A.D. MacKerell, J.B. Klauda, and W. Im. 2016. <a href="https://doi.org/10.1021/acs.jctc.5b00935"><span>CHARMM-GUI Input Generator</span> for <span>NAMD</span>, <span>GROMACS</span>, <span>AMBER</span>, <span>OpenMM</span>, and <span>CHARMM</span>/<span>OpenMM Simulations Using</span> the <span>CHARMM36 Additive Force Field</span></a>. <em>Journal of Chemical Theory and Computation</em>. 12:405–413.</div>
</div>
<div id="ref-hooverCanonicalDynamicsEquilibrium1985" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Hoover, W.G. 1985. <a href="https://doi.org/10.1103/PhysRevA.31.1695">Canonical dynamics: <span>Equilibrium</span> phase-space distributions</a>. <em>Physical Review A</em>. 31:1695–1697.</div>
</div>
<div id="ref-noseUnifiedFormulationConstant1984" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Nosé, S. 1984. <a href="https://doi.org/10.1063/1.447334">A unified formulation of the constant temperature molecular dynamics methods</a>. <em>The Journal of Chemical Physics</em>. 81:511–519.</div>
</div>
<div id="ref-parrinelloPolymorphicTransitionsSingle1981" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Parrinello, M., and A. Rahman. 1981. <a href="https://doi.org/10.1063/1.328693">Polymorphic transitions in single crystals: <span>A</span> new molecular dynamics method</a>. <em>Journal of Applied Physics</em>. 52:7182–7190.</div>
</div>
<div id="ref-ggplot" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Wickham, H. 2016. Ggplot2: <span>Elegant</span> graphics for data analysis. <span>Springer-Verlag New York</span>.</div>
</div>
<div id="ref-blender" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Community, B.O. 2018. Blender - a <span>3D</span> modelling and rendering package.</div>
</div>
<div id="ref-vmd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Humphrey, W., A. Dalke, and K. Schulten. 1996. <span>VMD</span> <span>Visual Molecular Dynamics</span>. <em>Journal of Molecular Graphics</em>. 14:33–38.</div>
</div>
<div id="ref-quarto" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Allaire, J.J., C. Teague, C. Scheidegger, Y. Xie, and C. Dervieux. 2022. <a href="https://doi.org/10.5281/zenodo.5960048">Quarto</a>.</div>
</div>
<div id="ref-knitr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Xie, Y. 2015. Dynamic documents with <span>R</span> and knitr. Second. <span>Boca Raton, Florida</span>: <span>Chapman and Hall/CRC</span>.</div>
</div>
<div id="ref-rbetterposter" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Aden-Buie, G. 2022. Betterposter: <span>A</span> better scientific poster.</div>
</div>
<div id="ref-chinthalapudiInteractionTalinCell2018a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Chinthalapudi, K., E.S. Rangarajan, and T. Izard. 2018. <a href="https://doi.org/10.1073/pnas.1806275115">The interaction of talin with the cell membrane is essential for integrin activation and focal adhesion formation</a>. <em>Proceedings of the National Academy of Sciences of the United States of America</em>. 115:10339–10344.</div>
</div>
</div>
</section>
</main>
<!-- /main column -->
<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
    } else {
      disableStylesheet(alternateStylesheets);
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    setTimeout(function() {
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->


</body></html>